data/test/scenarios/wml_tests/ScenarioWML/EventWML/ActionWML/DirectActionsWML/kill_fires_events.cfg

```diff
@@ -0,0 +1,34 @@
+# This test checks if the last_breath and die events are triggered when a unit on the recall list is killed with [kill] tag
+# The default behavior is that only units on the map trigger the events, so this test will fail with alice_breath and alice_die

```

Comment:
There's a standard format used in all the other tests for documenting what the test does. For this one for example, it'd look like:
```
#####
# API(s) being tested: [kill]fire_event=yes
##
# Actions:
# Kill bob and alice.
##
# Expected end state:
# A last breath event and a die event is triggered for both bob and alice.
#####
```